docs/userguide/networking/get-started-overlay.md

```diff
@@ -91,7 +93,7 @@ that machine options that are needed by the `overlay` network driver.
 
 		$ docker-machine create \
 		-d virtualbox \
-		--swarm --swarm-image="swarm" --swarm-master \
+		--swarm --swarm-master \
```

Comment:
oh, sorry, I commented that on the original PR, wasn't aware you did it on purpose. (Not sure having them on a separate line makes it harder to copy though)